diagram-viewer/src/main/java/com/powsybl/diagram/viewer/MainViewController.java

```diff
@@ -380,6 +456,23 @@ private void initSubstationsTree(Network network) {
         vlTree.setShowRoot(true);
     }
 
+    private boolean testComponent(ComponentFilterType type, Identifiable<?> identifiable) {
```

Comment:
We only call this for voltage levels and substations, so we can be more precise than the identifiable class. The name can also be more explicit. And the method can be static.
```suggestion
    private static boolean containsComponentType(ComponentFilterType type, Container<?> container) {
```